robottelo/constants/__init__.py

```diff
@@ -18,7 +18,7 @@ class Colored(Box):
 # This should be updated after each version branch
 SATELLITE_VERSION = "6.15"
 SATELLITE_OS_VERSION = "8"
-SAT_NON_GA_VERSIONS = ['6.14', '6.15']
+SAT_NON_GA_VERSIONS = ['6.15', '6.16']
```

Comment:
This is used mainatain sat and capsule repo setup tests, it contains a check for non-GA version repo check, keeping it won't harm but it will be unnecessary check for 6.15.z, right? @jameerpathan111 @Griffin-Sullivan  
```suggestion
SAT_NON_GA_VERSIONS = ['6.15']
```

Reply:
@jameerpathan111 @Griffin-Sullivan @Gauravtalreja1 There is condition based on this constant in `test_advanced.py` even that has to be removed and cherrypicked ?

Should I just remove this change from this PR and later someone will raise a followup PR for it ?

Reply:
I think just removing 6.16 from constants should be fine